pkgs/tools/security/chntpw/default.nix

```diff
@@ -11,11 +11,47 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ unzip ];
-  buildInputs = lib.optionals stdenv.isLinux [ stdenv.glibc.out stdenv.glibc.static ];
 
   patches = [
     ./00-chntpw-build-arch-autodetect.patch
     ./01-chntpw-install-target.patch
+    # Import various bug fixes from debian
+    (fetchpatch {
+      url = "https://sources.debian.org/data/main/c/chntpw/140201-1/debian/patches/04_get_abs_path";
+      sha256 = "0sfakwi8vpjwpnknxm8a2cyj1jwazdarrzvwnbhzql1gbxpvnry7";
```

Comment:
Please don't generate the URLs with prefetch-url because that misses some cleanup done by fetchpatch. See https://logs.nix.ci/?key=nixos/nixpkgs.169560&attempt_id=901bfdd1-2425-46e0-a2a8-768a5fafede4